### DISCUSSION TITLE
Status of entities by hovering over with the mouse

### DISCUSSION THREAD

**Discussion by mcv93us** [ExperienceLovelace/ha-floorplan#126] [Bugs] [answered]
Hi, with the latest version of HA Floorplan v1.0.24 when I hover the mouse over an entity mapped in the SVG it no longer shows me the colored outline of the entity and not even the status information but only the title I assigned to the entity appears when I edited it on Inkscape.
With version 1.0.23 everything works as before; what was changed that broke compatibility?
I use Floorplan on a docker installation of Home Assistant 2021.6.6 for Raspberry PI 4 and connect to the interface via the web with Chrome 91.0 on Windows 10.

**Comment by pkozul** [comment, marked as answer]
Hi @mcv93us 
Thanks for letting us know. Yes, the latest release does contain an optimisation for hover-over. In previous versions, every time any of the entities changed, floorplan would update the hover-over text of all entities on the floorplan. With this latest version, the hover-over text is only updated when you actually hover over an entity. We tested this, but it could be that your scenario is different to what we tested. Can you paste your floorplan SVG (even a cut-down, simple version of it). the relevant floorplan config YAML, and the CSS you use for the hover-over effect?

**Reply by mcv93us** [reply]
Ok, I have attached the reduced version of the SVG without background plan but only with the entities (I renamed the resources of the objects of course), the .css file and the .yaml configuration, all in a zipped file, I hope it will suffice.
With version 1.0.24 no information appears if I pass over with the mouse while with version 1.0.23 I have all the necessary information and the object is outlined.
[my_floorplan.zip](https://github.com/ExperienceLovelace/ha-floorplan/files/6765694/my_floorplan.zip)

**Reply by pkozul** [reply]
Hi @mcv93us 

We have fixed the bug, and the new version (1.0.25) will soon be released to HACS. We will let you know when it's ready.

**Reply by exetico** [reply]
It's now released.

https://github.com/ExperienceLovelace/ha-floorplan/releases/tag/1.0.25 

Thanks for the report, and happy updating in HACS!

_I've updated the category._

**Reply by mcv93us** [reply]
Thanks a lot, it works perfectly!
